### ISSUE TITLE
add a Kubernetes operator for reading quotas from a k8s TPR

### ISSUE THREAD

**Issue by majewsky** [sapcc/limes#15]
Register a Kubernetes TPR (third-party resource) for Limes quotas. An instance of it could look something like this:

```yaml
clusters:
- id: first
  domains:
  - name: foo
    projects:
    - name: bar
      services:
      - type: compute
        resources:
        - name: cores
          quota: 10
```

The operator would then enforce these quotas in the Limes database and in the backend. Also, it would set a flag on the affected project or domain that forbids manual quota management via `PUT` requests.

**Comment by majewsky** [comment]
A less cloudy variant of this is implemented by #42.

**Comment by majewsky** [comment]
Going to close this. Quota seeds have evolved into quota constraints which do this and more.
